client/assets/images/customer-home/illustration--titan-banner.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="225" height="134" viewBox="0 0 225 134" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we remove this file? Seeing as we've switched to an existing image, I don't think we need this one!